lib/models/device.coffee

```diff
@@ -389,7 +381,11 @@ getDeviceModel = (deps, opts) ->
 	# });
 	###
 	exports.getApplicationName = (uuidOrId, callback) ->
-		exports.get(uuidOrId, select: 'application_name').get('application_name').asCallback(callback)
+		exports.get uuidOrId,
+			expand: 'application': $select: 'app_name'
```

Comment:
You can drop the quotes from `'application'`

Reply:
Done